core/src/OAuth2/OAuth2Client.cs

```diff
@@ -94,7 +90,7 @@ public void AddBodyParameter(string key, string value)
             HttpResponse response = null;
             Uri endpointUri = CreateFullEndpointUri(endPoint);
             var httpEvent = new HttpEvent() { HttpPath = endpointUri, QueryParams = endpointUri.Query };
-            Client.Telemetry.GetInstance().StartEvent(requestContext.TelemetryRequestId, httpEvent);
+            Telemetry.Telemetry.GetInstance().StartEvent(requestContext.TelemetryRequestId, httpEvent);
```

Comment:
>Telemetry.Telemetry [](start = 12, length = 19)

while correct, it does look odd... consider re-using the alias from above to point to Telementry.Telemetry ... or perhaps the Telemetry class should be named differently